frontend/src/Components/Chart/CartesianCharts/utils/types.ts

```diff
@@ -0,0 +1,49 @@
+export type CartesianChartsData = {
+  label: string;
+  value: number;
+};
+
+export type CartesianChartProps = {
+  chartTitle: string;
+  data: CartesianChartsData[];
+  size: 'small' | 'medium' | 'large' | 'xlarge';
+  xAxisLegend: string;
+  yAxisLegend: string;
+  yLabelCount: number;
+  hasXDirLines?: boolean;
+  hasYDirLines?: boolean;
+  splitYLabel?: [number, number];
+  splitXLabel?: [number, number];
+};
+
+export type CartesianChartsSize = {
+  cWith: number;
+  cHeight: number;
+  labelFont: number;
+  topPadding: number;
+  rightPadding: number;
+  bottomPadding: number;
+  leftPadding: number;
+  xLabelsMargin: number;
+  yLabelsPosition: number;
+};
+
+export type CartesianChartSizes = {
+  small: CartesianChartsSize;
+  medium: CartesianChartsSize;
+  large: CartesianChartsSize;
+  xlarge: CartesianChartsSize;
+};
+
+export type CartesianChartsColors = {
+  bar: string;
+  bg: string;
+  text: string;
```

Comment:
Syns det er tydeligere med dette

```suggestion
  background: string;
  foreground: string;
```